cli/cmd/upgrade.go

```diff
@@ -201,6 +199,9 @@ func upgradeRunE(options *upgradeOptions, stage string, flags *pflag.FlagSet) er
 		fmt.Fprintf(os.Stderr, "%s\n", controlPlaneMessage)
 	}
 
+	fmt.Fprintf(os.Stderr, "\n")
```

Comment:
This will push a newline into stderr even if there are no messages; can you please make it so that only happens if there was output?